extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -352,21 +362,26 @@ public function combinedMediaSearch() {
 			->setLimit( 4 )
 			->setStart( $this->getVal( 'next', 0 ) );
 
-		$results = $this->queryServiceFactory->getFromConfig( $config )->searchAsApi( [ 'url', 'id', 'pageid', 'wid', 'title' ], true );
+		$results = $this->queryServiceFactory->getFromConfig( $config )->searchAsApi(
+			[ 'url', 'id', 'pageid', 'wid', 'title' ],
+			true
+		);
 		$dimensions = [ 'width' => 120, 'height' => 90 ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "height" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "width" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)